models/issues/pull.go

```diff
@@ -448,6 +448,23 @@ func (pr *PullRequest) GetGitHeadBranchRefName() string {
 	return fmt.Sprintf("%s%s", git.BranchPrefix, pr.HeadBranch)
 }
 
+func (pr *PullRequest) GetReviewCommentsCount(ctx context.Context) int {
```

Comment:
At best, define here again what counts as a review comment, i.e. in the method comment.

Reply:
I added a discriptive comment